src/pages/home/ReportScreen.js

```diff
@@ -199,6 +215,16 @@ class ReportScreen extends React.Component {
         Navigation.navigate(ROUTES.getReportRoute(this.props.accountManagerReportID));
     }
 
+    /**
+     * When the app is visible and the LHN is not opening in small-screen devices we can assume that the report is fully visible.
+     *
+     * @returns {Boolean}
+     */
+    isReportFullyVisible() {
+        const isSidebarCoveringReportView = this.props.isSmallScreenWidth && this.props.isDrawerOpen;
+        return Visibility.isVisible() && !isSidebarCoveringReportView;
+    }
```

Comment:
Hmm looks like this function is a copy of what we already have in ReportActionsView right? Should we make a lib and use this in both places?

Reply:
What do you think you'd pass as a prop to ReportActionsView? 

And we don't necessarily need to think about "adding more logic in the future" because we can easily change things at that point - for now lets figure out what's the best current solution.

@mananjadhav what do you think about this, do you share my concern about 2 functions in different files that are basically identical?

Reply:
Ok cool thanks for the discussion!

And yes you're exactly correct about putting these types of reusable libs into the `src/libs` folder 👍

Reply:
just went through the discussion, and yeah I would've agreed to remove any duplicate code. apologies couldn't get to this earlier.